students/taeyeon/users/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json,re
 
-# Create your views here.
+from django.http            import JsonResponse
+from django.views           import View
+from django.core.exceptions import ValidationError
+
+from .models                import User
+
+class UsersView(View) :
+    def post(self,request) :
+        try: 
+            data           = json.loads(request.body)
+            email          = data['email']
+            password       = data['password']
+            regex_email    = r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            regex_password = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
```

Comment:
정규표현식은 문자열이기 때문에 상수로 선언해주시고, UserView class의 member 변수로 선언해서 다른 hanlder method에서도 사용할 수 있도록 하며 좋을 것 같습니다!

```suggestion
class UsersView(View) :
    REGEX_EMAIL  = r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
    REGEX_PASSWORD = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'

    def post(self,request) :
        try: 
            data           = json.loads(request.body)
            email          = data['email']
            password       = data['password']
```